pr2eus/pr2-interface.l

```diff
@@ -324,6 +325,82 @@ Example: (send self :gripper :rarm :position) => 0.00"
 	 (when (and current origin)
 	   (mapcar #'v- current origin))
        current)))
+  (:switch-controller
+   (stop start &key (strictness 1))
+   (let (req res start-controller stop-controller)
+     (ros::wait-for-service "/pr2_controller_manager/switch_controller")
+     (setq req (instance pr2_mechanism_msgs::SwitchControllerRequest :init))
+     (setq start-controller (cond (start start) (t "dummy_controller")))
+     (setq stop-controller (cond (stop stop) (t "dummy_controller")))
+     (case stop-controller
```

Comment:
@h-kamada Is there any reference to decide this name conversion rule? (e.g. `:head-traj` -> `"head_traj_controller"`)
If not, I recommend this function take arguments controller name as just string instead of keyword because we can know what type of controller are available by calling `:list-controllers` method, right?